contracts/proposal/cwd-proposal-multiple/src/testing/adversarial_tests.rs

```diff
@@ -229,3 +235,138 @@ fn test_execute_proposal_more_than_once() {
         .unwrap();
     assert!(matches!(err, ContractError::NotPassed {}));
 }
+
+// Users should be able to submit votes past the proposal
+// expiration date. Such votes do not affect the outcome
+// of the proposals; instead, they are meant to allow
+// voters to voice their opinion.
+#[test]
```

Comment:
this is a great test. love this docstring and the comments inside to walk through what it's doing